Xamarin.Forms.Core/Items/GroupableItemsView.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Xamarin.Forms
+{
+	public class GroupableItemsView : SelectableItemsView
+	{
+		public static readonly BindableProperty IsGroupingEnabledProperty =
```

Comment:
I feel , as often, that we're doing too much when naming bool properties. there's some redundancy in this name, and `GroupingEnabled` or `IsGrouped` carries as much information...

feel free to dismiss this opinion

Reply:
Agreed, and changed to IsGrouped (which more accurately reflects the behavior).